docs/source/using/image-caching.adoc

```diff
@@ -4,39 +4,121 @@ include::variables.adoc[]
+Exporting <image-name-0>. OK
+Pulling image <image-name-1> .. OK
+Exporting <image-name-2>. OK
 ----
 
-Once enabled, caching occurs transparently, in a background process, the first time you use the xref:../command-ref/minishift_start#[`minishift start`] command.
-Once the images are cached under *_$MINISHIFT_HOME/cache/images_*, successive {project} VM creations will use these cached images.
+[NOTE]
+====
+Images which are not available in the Docker daemon will be pulled prior to be exported to the host.
+====
 
-Each time an image exporting background process runs, a log file is generated under *_$MINISHIFT_HOME/logs_* which can be used to verify the progress of the export.
+To import a previously cached images use the xref:../command-ref/minishift_image_import.adoc#[`minishift image import`] command:
 
-You can disable the caching of the OpenShift images by setting `image-caching` to `false` or removing the setting altogether using xref:../command-ref/minishift_config_unset#[`minishift config unset`]:
+----
+$ minishift image import <image-name-0> <image-name-1> ...
+Importing <image-name-0> . OK
+----
+
+[[listing-images]]
+=== Listing Images
+
+The xref:../command-ref/minishift_image_list.adoc#[`minishift image list`] commands lists either the currently cached images or the images available in the {project} Docker daemon.
+
+To view currently cached images on the host:
 
 ----
-$ minishift config unset image-caching
+$ minishift image list
+openshift/origin-docker-registry:v3.6.0
+openshift/origin-haproxy-router:v3.6.0
+openshift/origin:v3.6.0
+----
+
+To view images available in the Docker daemon:
+
+----
+$ minishift image list --vm
+openshift/origin-deployer:v3.6.0
+openshift/origin-docker-registry:v3.6.0
+openshift/origin-haproxy-router:v3.6.0
+openshift/origin-pod:v3.6.0
+openshift/origin:v3.6.0
+----
+
+[[persisting-image-names]]
+=== Persisting Cached Image Names
+
+In order to avoid having to type the image names explicitly as part of the `image export` and `image import` command, you can persist the default image names for import and export in the persistent configuration:
+
+----
+$ minishift config set cache-images <image-name-0>,<image-name-1>
 ----
 
+Once the image names are stored in the persistent configuration, you can run `minishift image export` and `minishift image import` without any arguments.
+
+[[all-images]]
+=== Exporting and Importing All Images
+
+Last but not least, you can export and import all images using the `--all` flag.
+For the export command this means that all images currently available on the Docker daemon will be exported to the host.
+For the import command it means that all images available in the local {project} cache will be imported into the Docker daemon of the {project} VM.
+
+[WARNING]
+====
+Exporting and importing all images can take a long time and locally cached images can take up a considerable amount of disk space. We recommend to use this feature cautiously.
```

Comment:
"We recommend using this feature with caution."

Also, one sentence per line, please.

Reply:
> Also, one sentence per line, please.

Of course